src/System.Private.CoreLib/shared/System/String.Comparison.cs

```diff
@@ -269,7 +265,7 @@ public static int Compare(string? strA, string? strB, CultureInfo culture, Compa
         // The case-sensitive option is set by ignoreCase, and the culture is set
         // by culture
         //
-        public static int Compare(string? strA, string? strB, bool ignoreCase, CultureInfo culture)
+        public static int Compare(string? strA, string? strB, bool ignoreCase, CultureInfo? culture)
```

Comment:
Same for all of these that delegate to that one.

Reply:
Still waiting for final decision in above discussion.